ruby-on-rails/docs/gem-tutorials/spree.md

```diff
@@ -0,0 +1,324 @@
+# How to use Spree gem
+
+*Abstract: In this documents, we will present some tricks to deal with spree gem. These tricks will be at nivel database, backoffice, backend, frontend... This will be a good base to start with spree and have a common way to work with this gem.*
+
+## Introduction
+
+Spree is a useful gem for Ruby On Rails, that allow to create an e-commerce in few command and a few configuration. Spree is composite of various gem and it accepts lots of extension gems developed by the community.
```

Comment:
allows*

Reply:
"an e-commerce in few command and a few configuration" can be "an e-commerce *with a* few commands and  configuration"

Reply:
"various gem*s*"